server/services/v1/query_runner.go

```diff
@@ -537,6 +537,7 @@ func (runner *SearchQueryRunner) Run(ctx context.Context, tx transaction.Tx, ten
 	}
 
 	facets, err := qsearch.UnmarshalFacet(runner.req.Facet)
+	// test if facet field is actually a schema field
```

Comment:
Will include this in another iteration, wanted to get server in sync with latest API